### PR TITLE
Move remaining inventory from 4.10 SNO to 4.12 compact

### PR DESCRIPTION
**Pull request by Sandeepyadav93**

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/hold

we need a PR to move jobs from openstack-operator/install_yaml to 4.12 first

**Comment by Sandeepyadav93** [comment]
We need to wait for https://github.com/openshift/release/pull/41636 to merge first

File: clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml [hunk 1]
```diff
@@ -37,17 +44,17 @@ spec:
   - name: openstack-k8s-operators-cdc-vexx-23
   labels:
     tp.openshift.io/owner: openstack-k8s-operators
-  maxConcurrent: 14
-  maxSize: 14
+  maxConcurrent: 21
+  maxSize: 21
```
Comment (marios) [inline clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml:48]: how are we deciding these values

Comment (Sandeepyadav93) [reply clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml]: @marios 

maxSize: We determine size from our inventory size, we have created  floating ip manually/entry in AWS and created CDC files for 20 clusters: https://github.com/openshift/release/blob/a8ec2ee966b50045082d6888172483810da1798f/clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-cdc-vexx-01.yaml ) for 20 OCP clusters.

~~~
  - name: openstack-k8s-operators-cdc-vexx-01
  - name: openstack-k8s-operators-cdc-vexx-02
  - name: openstack-k8s-operators-cdc-vexx-03
  - name: openstack-k8s-operators-cdc-vexx-04
  - name: openstack-k8s-operators-cdc-vexx-05
  - name: openstack-k8s-operators-cdc-vexx-06
  - name: openstack-k8s-operators-cdc-vexx-07
  - name: openstack-k8s-operators-cdc-vexx-08
  - name: openstack-k8s-operators-cdc-vexx-09
  - name: openstack-k8s-operators-cdc-vexx-10
  - name: openstack-k8s-operators-cdc-vexx-11
  - name: openstack-k8s-operators-cdc-vexx-12
  - name: openstack-k8s-operators-cdc-vexx-13
  - name: openstack-k8s-operators-cdc-vexx-14
  - name: openstack-k8s-operators-cdc-vexx-15
  - name: openstack-k8s-operators-cdc-vexx-16
  - name: openstack-k8s-operators-cdc-vexx-17
  - name: openstack-k8s-operators-cdc-vexx-18
  - name: openstack-k8s-operators-cdc-vexx-19
  - name: openstack-k8s-operators-cdc-vexx-20
  ~~~
  
  runningCount is how many clusters we want in ready state as standby(ready to be claimed anytime)

Comment (marios) [reply clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-12-vexxhost-hybrid_clusterpool.yaml]: thank you for the pointers @Sandeepyadav93

**Comment by viroel** [comment]
/hold cancel

all jobs are now configured for 4.12

**Comment by viroel** [comment]
last ones that were moved to 4.12:
https://github.com/openshift/release/pull/41636

**Pull request merged**